packages/orbit-components/src/Select/index.tsx

```diff
@@ -62,6 +63,11 @@ const Select = React.forwardRef<HTMLSelectElement, Props>((props, ref) => {
   const inputRef = React.useRef<HTMLLabelElement | null>(null);
 
   const shown = tooltipShown || tooltipShownHover;
+  const tooltipId = shown ? `${selectId}-feedback` : undefined;
+
+  const ariaDescribedbyValue = insideInputGroup
+    ? ariaDescribedby
+    : [ariaDescribedby, tooltipId].filter(Boolean).join(" ") || undefined;
```

Comment:
I'm wondering if this will work nicely for Right To Left languages. Is that a pattern you've already applied elsewhere?

Reply:
We are using similar patter [here in the Seat](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/Seat/index.tsx#L49).

I'll ad this to our document and create task for us to revisit RTL patterns in the Phase 3 :+1: